lib/dagDisplay.ml

```diff
@@ -865,20 +865,26 @@ let set_vother x = Vother x
 
 (* TODO should vl be Vint list and f = max|min, not a string?*)
 let eval_predefined_apply f vl =
-  let vl = List.map (function VVstring s -> s | _ -> raise Not_found) vl in
+  let vl =
+    List.map
+      (function
+        | VVstring "" -> (
+            match f with
+            | "min" -> max_int
+            | "max" -> 0 - max_int
```

Comment:
I'm not sure what you're trying to achieve, but `0 - max_int` looks really suspicious... You have `min_int` available.

Reply:
Thanks for the tip!!